sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachineDiskOptions.java

```diff
@@ -15,41 +15,67 @@ public final class VirtualMachineDiskOptions {
     private DeleteOptions deleteOptions;
     private Boolean writeAcceleratorEnabled;
 
+    /**
+     * Creates a new instance of VirtualMachineDiskOptions.
+     */
+    public VirtualMachineDiskOptions() {
+    }
+
     // DiskEncryptionSetParameters instance without ID means do not configure.
     // If disk is already encrypted with CMK, it remains so when attached.
     private DiskEncryptionSetParameters diskEncryptionSetOptions;
 
-    /** @return the disk storage account type */
+    /**
+     * Gets the disk storage account type.
+     *
+     * @return the disk storage account type
+     */
     public StorageAccountTypes storageAccountType() {
         return storageAccountType;
     }
 
-    /** @return the disk caching type */
+    /**
+     * Gets the disk caching type.
+     *
+     * @return the disk caching type
+     */
     public CachingTypes cachingTypes() {
         return cachingTypes;
     }
 
-    /** @return the disk delete options */
+    /**
+     * Gets the disk delete options.
+     *
+     * @return the disk delete options
+     */
     public DeleteOptions deleteOptions() {
         return deleteOptions;
     }
 
     /**
-     * Gets whether the write accelerator is enabled. {@literal null} if no change.
+     * Whether the write accelerator is enabled. {@literal null} if no change.
      *
      * @return whether the write accelerator is enabled
      */
     public Boolean writeAcceleratorEnabled() {
         return writeAcceleratorEnabled;
     }
 
-    /** @return whether disk encryption set is configured,
-     * either as the ID of disk encryption set, or as {@code null} to override default configuration. */
+    /**
+     * Whether disk encryption set is configured.
```

Comment:
Checks

Reply:
Fixed by the [commit#6d08161](https://github.com/Azure/azure-sdk-for-java/pull/43499/commits/6d0816182ab673e2c8f296b4c2f6c0ba843f2867) and [commit#2570846](https://github.com/Azure/azure-sdk-for-java/pull/43499/commits/25708467caa912ea75c4cdbfe2e9e1f06b7bee25)